.github/workflows/ci.yml

```diff
@@ -74,3 +74,99 @@ jobs:
 
       - name: Build project
         run: sbt ++${{ matrix.scala }} test 'scalaParser/testOnly scalaparser.SnippetSpec'
+
+      - name: Compress target directories
+        run: tar cf targets.tar examples/target parboiled-core/.native/target target scalaParser/target jsonBenchmark/target parboiled/.js/target parboiled/.native/target parboiled/.jvm/target parboiled-core/.jvm/target parboiled-core/.js/target project/target
+
+      - name: Upload target directories
+        uses: actions/upload-artifact@v2
+        with:
+          name: target-${{ matrix.os }}-${{ matrix.scala }}-${{ matrix.java }}
+          path: targets.tar
+
+  publish:
+    name: Publish Artifacts
+    needs: [build]
+    if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/master')
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        scala: [3.2.2]
+        java: [temurin@8]
```

Comment:
Incase its not clear, this determines the JDK version when making a release (which in this case is JDK 8)

Reply:
Thank you!